samples/test/features/security-questions.feature

```diff
@@ -0,0 +1,45 @@
+Feature: Security Questions
+
+  Background:
+    Given a Policy that defines "Profile Enrollment"
+      And with a Policy Rule that defines 'collecting default attributes and a required "Age" and emailVerification is not required'
+      And a Policy that defines "MFA Enrollment" with properties
+        | okta_password     | REQUIRED |
+        | security_question | REQUIRED |
+      And with a Policy Rule that defines "MFA Enrollment Challenge"
+      And a user named "Mary"
+      And she does not have account in the org
+
+  Scenario: Mary signs up for an account and enrolls in Password and a custom Security Question
+    When she clicks the 'signup' button
+    Then she is redirected to the "Self Service Registration" page
+    When she fills out her First Name
+      And she fills out her Last Name
+      And she fills out her Email
+      And she fills out her Age
+      And she submits the form
+    Then she is redirected to the "Select Authenticator" page
+    When she selects the "Password" factor
+      And she submits the form
+    Then she sees the set new password form
```

Comment:
We can use the `she is redirected to "xxx" page` pattern to keep steps consistence and discoverable.